druid-shell/src/keyboard.rs

```diff
@@ -84,6 +115,19 @@ impl KeyEvent {
         }
     }
 
+    /// Test whether the key matches the given pattern.
+    ///
+    /// # Examples
+    /// ```
+    /// use druid_shell::{KeyCode, Modifier};
+    /// # let key_event = druid_shell::KeyEvent::for_test(Modifier::NoMods, "a", KeyCode::KeyA);
+    /// let is_undo = key_event.matches(Modifier::Ctrl + 'z');
+    /// let is_reset = key_event.matches(Modifier::Ctrl + Modifier::Alt + KeyCode::Delete);
```

Comment:
In general, I think it's important in our example code to show how things are supposed to work cross-platform; the first of these examples excludes the mac, and the second is windows only?

Reply:
Oh, I did the "Cmd" thing after writing the docstring. That does the platform-specific behavior you asked.